bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -128,17 +128,14 @@ export function isAllowedTo(action, credentials) {
     userId,
   });
 
-  const allowedToInitiateRequest =
-    user &&
+  const allowedToInitiateRequest = user &&
     user.authToken === authToken &&
     user.validated &&
     user.clientType === 'HTML5' &&
     user.user &&
     user.user.connection_status === 'online';
 
-  const listOfSafeActions = [
-    'logoutSelf',
-  ];
+  const listOfSafeActions = new Array('logoutSelf');
```

Comment:
There is an interesting [benchmark ](https://jsperf.com/literal-vs-new-23) on different ways of how you can initialize an array. I think it doesn't matter for us here at all since we initialize it just once, but it's just good to know :-)